src/utils/tool/random.go

```diff
@@ -3,15 +3,10 @@ package tool
 import (
 	crypto_rand "crypto/rand"
 	"math/rand"
-	"time"
 )
 
 const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
 
-func init() {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Why? Without this seed is always 1.

Reply:
In Go starting from version `1.20`, the global random number generator is automatically seeded.
https://tip.golang.org/doc/go1.20